README.md

```diff
@@ -1,2 +1,270 @@
+
+### /medicationClasses/$medicationClassId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|e.g. "BETABLOCKER"||
+|name|LocalizedText|-|A name for a given medicationClass to be displayed to a user.|
+|videoPath|string|e.g. "/videoSectionId/1/videos/2"|The path to retrieve the respective video from Firestore.|
+
+## /videoSections/$videoSectionId$
+
+In this section, we describe all data related to educational videos to be shown in the Engage-HF mobile apps. The videos are grouped into different categories to be displayed as sections in the mobile apps.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "ENGAGE-HF Application"|May be localized.|
+|description|LocalizedText|e.g. "Helpful videos on the ENGAGE-HF mobile application."|May be localized. Are there different videos / videoSections for each platform?|
+|orderIndex|integer|e.g. 1|Since Firestore collections aren't necessarily ordered, we have this property to order the elements by on the clients. The list is supposed to be ordered ascending by `orderIndex`.|
+
+### /videoSections/$videoSectionId$/videos/$videoId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "Beta Blockers for Heart Failure"|May be localized.|
+|url|LocalizedText|e.g. "https://youtu.be/XfgcXkq61k0"|May be localized.|
+|orderIndex|integer|e.g. 1|Since Firestore collections aren't necessarily ordered, we have this property to order the elements by on the clients. The list is supposed to be ordered ascending by `orderIndex`.|
+
+## /users/$userId$
+
+In this section, we describe all user-related data to be stored. The security rules shall be set up in a way to only allow for a patient to access its own information and a clinician to access all patients' information (or even more restrictive, if needed).
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|dateOfBirth|Date|-|To be used for verification purposes.|
+|invitationCode|string|-|The invitationCode to be used when logging in to the app for the first time.|
+|language|optional string|e.g. "en"|Following IETF BCP-47 / [FHIR ValueSet languages](https://hl7.org/fhir/R4B/valueset-languages.html)|
+|dailyRemindersAreActive|boolean|true, false|Decides whether to send out daily reminder messages for this user.|
+|textNotificationsAreActive|boolean|true, false|Decides whether to send text notifications for this user.|
+|medicationRemindersAreActive|boolean|true, false|Decides whether to send medication reminder messages for this user.|
+|timeZone|string|e.g. "America/Los_Angeles"|The value needs to correspond to an identifier from [TZDB](https://nodatime.org/TimeZones). It must not be an offset to UTC/GMT, since that wouldn't work well with daylight-savings (even if there is no daylight-savings time at that location). Also, don't use common abbreviations like PST, PDT, CEST, etc (they may be ambiguous, e.g. CST).|
+
+TBD: What happens if the timeZone could not be identified? When would we send out messages?
```

Comment:
I think we would should assume `America/Los_Angeles` for now if nothing has been set.

Reply:
✅  added as part of the comment column and removed the TBD

Reply:
Thank you!